reference/service/permissions.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Permissions
+---
+
+The Pulumi Cloud Console provides fine-grained access controls for stacks.
+
+Pulumi stacks are "owned" by an organization, and the first level of access
```

Comment:
Probably worth clarifying that these options are only available for Organization-owned stacks, and not User-owned ones.

Reply:
Done.